### PR TITLE
St 439

### PR DESCRIPTION
**Pull request by lewey**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala [hunk 1]
```diff
@@ -51,21 +61,26 @@ case class CheckYourAnswersViewModel(paymentFrequency: String,
...
+      messages(dynamicTitle(Some(paymentFrequency), payPeriodInDays, taxablePayMessages)),
       taxablePay,
       controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.taxablePayslipAmountPage().url,
       isMonetaryValue
     )
 
     val hasBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
-      messages("tai.estimatedPay.update.checkYourAnswers.hasBonusOrOvertime",currentTaxYearRangeHtmlNonBreakBetween),
+      messages("tai.estimatedPay.update.checkYourAnswers.hasBonusOrOvertime",TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
       Some(messages(s"tai.label.${hasBonusOrOvertime.toLowerCase}")),
```
Comment (oc-ste) [inline app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala]: Think neda is wanting these to be currentTaxYearRangeSingleLine, we should probably update the deprecated line to reflect this

File: app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala [hunk 2]
```diff
@@ -51,21 +61,26 @@ case class CheckYourAnswersViewModel(paymentFrequency: String,
...
       controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.bonusPaymentsPage().url
     )
 
     val totalBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
-      messages("tai.estimatedPay.update.checkYourAnswers.totalYearlyBonusOrOvertime", currentTaxYearRangeHtmlNonBreak),
+      messages("tai.estimatedPay.update.checkYourAnswers.totalYearlyBonusOrOvertime", TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
       totalBonusOrOvertime,
```
Comment (oc-ste) [inline app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala]: Same as above

File: app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala (hunk 1 above)
Comment (lewey) [reply app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala]: I believe it is just headings

File: app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala (hunk 2 above)
Comment (lewey) [reply app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala]: I believe it is just headings

**Pull request merged**